### PR TITLE
feat: stripe

### PR DESCRIPTION
**Pull request by simonorzel26**

### CODE REVIEW

File: apps/web/package.json [hunk 1]
```diff
@@ -11,15 +11,18 @@
...
     "@next/font": "^13.0.6",
     "@prisma/client": "^4.5.0",
+    "@stripe/stripe-js": "^1.46.0",
     "@tanstack/react-query": "^4.16.0",
     "@trpc/client": "^10.0.0",
     "@trpc/next": "^10.0.0",
     "@trpc/react-query": "^10.0.0",
     "@trpc/server": "^10.0.0",
+    "@tryabby/react": "workspace:*",
+    "@types/stripe-v3": "^3.1.27",
```
Comment (cstrnt) [inline apps/web/package.json:24]: this should be a dev dependency

File: apps/web/package.json [hunk 2]
```diff
@@ -11,15 +11,18 @@
...
+    "csstype": "^3.0.10",
```
Comment (cstrnt) [inline apps/web/package.json:25]: do we need this?

File: apps/web/src/pages/api/checkout/index.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+		return stripePromise
+	}
+
+	const stripe = await getStripe()
+
+	await stripe?.redirectToCheckout({
+		mode: 'payment',
+		lineItems,
+		successUrl: `${window.location.origin}?session_id={CHECKOUT_SESSION_ID}`,
+		cancelUrl: window.location.origin
```
Comment (cstrnt) [inline apps/web/src/pages/api/checkout/index.ts:20]: I think it's misleading that this lives inside the `pages` folder

**Review by cstrnt — CHANGES_REQUESTED** [review]

File: apps/web/package.json (hunk 2 above)
Comment (simonorzel26) [reply apps/web/package.json:25]: its a warn peer dep

File: apps/web/src/pages/api/checkout/index.ts (hunk 3 above)
Comment (simonorzel26) [reply apps/web/src/pages/api/checkout/index.ts:20]: Where should it live?

**Pull request merged**